gems/quilt_rails/lib/quilt_rails/configuration.rb

```diff
@@ -11,6 +11,7 @@ def initialize
 
       self.react_server_host     = "#{react_server_ip}:#{react_server_port}"
       self.react_server_protocol = ENV['REACT_SERVER_PROTOCOL'] || "http"
+      self.logger                = ::Logger.new($stdout)
```

Comment:
looks weird to me to set this logger inside the configuration. `Quilt::Configuration.logger.log` :/ 

Why not do like Rails and just set a class variable in the Quilt module directly ?

Reply:
We already have a configuration object with a configuration pattern though. The new API is `Quilt.logger` which delegates to `Quilt.configuration.logger`. As long as the API is the same I don't see any problems.

Reply:
No problem indeed but since you are delegating `Quilt.logger` to the configuration I don't see the point of not adding that directly in the logger. That being said you also can't use `delegate` because that gem don't depend on active support

Reply:
> No problem indeed but since you are delegating Quilt.logger to the configuration I don't see the point of not adding that directly in the Quilt module. 

I mainly want to maintain the contract of the configuration object. We would have to deprecate `Quilt.configuration` too.

> That being said you also can't use delegate because that gem don't depend on active support

It depends on railties which depends on active support, but I could add a dependency in the gemspec to make that more apparent.